sale_commission_delivery_partner/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module solves inconsistency problems between SO and invoices
+regarding the selected agent commissioners.
+
+Installing this module will make sure to use the Delivery Address' agents instead
```

Comment:
This module addresses the use case where the same customer has different agents/commissions according to SO/Invoice delivery address.

In SO and invoice, commission is calculated based on agents set in the "delivery address" contact rather than on "customer" contact's agents.